.github/workflows/ci-workflow.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         with:
           fetch-depth: 0 # check out the entire repo history for SHA verification in lastSuccessful step
 
+      - name: Lint Markdown docs
+        run: make lint-markdown
+
       - name: Lint specification
         run: make lint
```

Comment:
NIT: Change to `make lint-apispec` in the top dir?

Reply:
Good point, changed!